packages/babel-parser/src/plugins/typescript.js

```diff
@@ -747,6 +753,22 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         this.next();
         return true;
       }
+
+      if (this.match(tt.braceL)) {
+        const braceStack = [tt.braceL];
+        this.next();
+
+        while (braceStack.length > 0) {
```

Comment:
Is it safe to skip the code between braces without parsing it?

e.g., wouldn't this loop accept something like `{bar: {}{foo = 2=>{} }}`?

Reply:
It does ignore everything, even if its invalid, but this function is only used in a lookahead() and it's job is to try to skip the first param after `(`.

it is used here: https://github.com/babel/babel/blob/be591698cd7238e8e5b8e682c7b82ad740527712/packages/babel-parser/src/plugins/typescript.js#L775

and this in turn is used here:
https://github.com/babel/babel/blob/be591698cd7238e8e5b8e682c7b82ad740527712/packages/babel-parser/src/plugins/typescript.js#L741

Reply:
Oh ok :+1: